mkdocs.yml

```diff
@@ -70,10 +70,17 @@ nav:
     - Installation: chapters/getting_started/installation.md
     - Contributing: chapters/getting_started/contributing.md
     - Troubleshooting: chapters/getting_started/troubleshooting.md
-  - Models and Data:
+  - PlantSeg Functionalities:
+    - XXX
+    - Data Processing: XXX
+    - Prediction: XXX
+    - Models: XXX
+    - Segmentation: XXX
```

Comment:
Is this intended?

Reply:
Those are only place holders, I will finalize them before merging